keras/engine/topology.py

```diff
@@ -1229,6 +1229,45 @@ def count_params(self):
                                    self.name + '.build(batch_input_shape)`.')
         return sum([K.count_params(p) for p in self.weights])
 
+    def count_flops(self):
```

Comment:
Another possibility is `count_flop` (without the `s` which may be confused for "per second").  I was concerned about confusion with `op` in TF just as @fchollet said that's why I didn't got for count_ops initially.  Happy to rename that to whatever you guys prefer.